docs/1_credits.html

```diff
@@ -12,7 +12,7 @@
 		<br>
 		<img src="functionalityRepresentation.png"></center>
 		<br>
-		Cancer Imaging Phenomics Toolkit (CaPTk) v.1.7.5.nonRelease.20191204 &emsp;|&emsp;<b>Contact:</b> software@cbica.upenn.edu
+		Cancer Imaging Phenomics Toolkit (CaPTk) v.1.7.5.nonRelease.20190111 &emsp;|&emsp;<b>Contact:</b> software@cbica.upenn.edu
```

Comment:
Can this be done via *.txt files?

Reply:
Technically, yes. Currently, the version information is embedded within the application on the CMake level as a pre-processor include. If we want to move away from this, we can put a "version" file somewhere in the repository and read that file's contents and display it when asked for the version (either on the CLI, GUI) and for Documentation, this would need to be picked up by Doxygen.